### PR TITLE
chore: changing more HPMN terminology

### PR DESCRIPTION
**Pull request by rmdash**
In all other dashpay/docs pages except dashmate

Preview build: https://dash-docs--263.org.readthedocs.build/en/263/

### CODE REVIEW

File: docs/user/masternodes/setup-evonode.rst [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, hpmn
+   :description: This guide describes how to set up a Dash evolution masternode. It also describes various options for hosting and different wallets
+   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, evonode
 
-.. _hpmn-setup:
+.. evonode-setup:
 
 ==========
-HPMN Setup
+EvoNode Setup
```
Comment (thephez) [inline docs/user/masternodes/setup-evonode.rst:8]: Header indicator (the "=" above an below) should be the same length as the text.

File: docs/user/masternodes/setup-evonode.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ more information.
 Before you begin
 ================
 
-This guide assumes you are setting up a single mainnet high-performance
+This guide assumes you are setting up a single mainnet evolution
```
Comment (thephez) [inline docs/user/masternodes/setup-evonode.rst:29]:
```suggestion
This guide assumes you are setting up a single mainnet Evolution
```

File: docs/user/masternodes/setup-evonode.rst [hunk 3]
```diff
@@ -65,7 +65,7 @@ in a configuration file and registration transaction as proof to write the
 configuration to the blockchain so the high-performance masternode can be
 included in the deterministic list.
 
-A high-performance masternode can be registered from the official Dash Core
+An evolution masternode (or EvoNode) can be registered from the official Dash Core
```
Comment (thephez) [inline docs/user/masternodes/setup-evonode.rst:68]:
```suggestion
An Evolution masternode (or EvoNode) can be registered from the official Dash Core
```

**Review by thephez — CHANGES_REQUESTED** [review]
:+1: Looks pretty good. Just a few things to clean up.

File: docs/user/masternodes/setup-evonode.rst (hunk 1 above)
Comment (rmdash) [reply docs/user/masternodes/setup-evonode.rst:8]: This is done

File: docs/user/masternodes/setup-evonode.rst [hunk 4]
```diff
@@ -1,11 +1,11 @@
 .. meta::
-   :description: This guide describes how to set up a Dash high-performance masternode. It also describes various options for hosting and different wallets
-   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, hpmn
+   :description: This guide describes how to set up a Dash evolution masternode. It also describes various options for hosting and different wallets
+   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, evonode
 
-.. _hpmn-setup:
+.. evonode-setup:
```
[the remark this answers is not in the document]
Comment (thephez) [reply docs/user/masternodes/setup-evonode.rst]: It looks like you accepted the suggested change, but I don't see the update requested in the comment.

File: docs/user/masternodes/setup-evonode.rst [hunk 5]
```diff
@@ -686,7 +686,7 @@ fee source** (``feeSourceAddress``). If you selected an external payout address,
 you must specify a fee source address. 
 
 Either the payout address or fee source address must have enough balance to pay
-the transaction fee, or the ``register_prepare_hpmn`` transaction will fail.
+the transaction fee, or the ``register_prepare_evonode`` transaction will fail.
```
Comment (thephez) [inline docs/user/masternodes/setup-evonode.rst:689]: I think all is good except these RPC names that got changed

**Pull request merged**